app/controllers/users_controller.rb

```diff
@@ -1,3 +1,30 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  rescue e
+    flash[:danger] = t "user_not_found"
+    redirect_to root_path
+  end
```

Comment:
```suggestion
  def show
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:danger] = t "user_not_found"
    redirect_to root_path
  end
```

nếu mún ... gọn hơn nữa
```suggestion
  def show
    return if @user = User.find_by id: params[:id]
    
    flash[:danger] = t "user_not_found"
    redirect_to root_path
  end
```
Chị suggest thay vì thế này mình check ifelse đi em. nó ổn định hơn để rescue